.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,28 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Shall we remove the IntelliJ related files(.idea/)

Reply:
done